roles/ceph-infra/tasks/setup_ntp.yml

```diff
@@ -45,7 +45,7 @@
 
     - name: enable chronyd
       service:
-        name: chronyd
+        name: chrony
```

Comment:
`chronyd` works fine with CentOS. Therefore, I suppose it's best to add a new task that runs only for Ubuntu.